app/src/main/java/com/github/geohunt/app/model/PointCalculator.kt

```diff
@@ -0,0 +1,73 @@
+package com.github.geohunt.app.model
+
+import com.github.geohunt.app.model.database.api.Location
+import kotlin.math.*
+
+/**
+ * Object giving access to its only method computePoints, which allows the user to compute
+ * the amount of points a claim gets.
+ */
+object PointCalculator {
+    /**
+     * Hyper parameters of the function used to compute the points
+     */
+    private const val MEAN = 0.0
```

Comment:
This parameter should be hard-coded because we don't want to change it. Being spot on should be equivalent to having all points !!

Reply:
I made the gaussian function expecting to move it in some MathUtils class later on. But I guess I could still hardcode the argument of the function to 0.0 without using a constant.